phpBB/styles/prosilver/theme/colours.css

```diff
@@ -44,6 +44,10 @@ a:hover {
 	color: #d41142;
 }
 
+a.topictitle-unread {
```

Comment:
do not append html tags in your css and please use proper component naming conventions for class names ie topic-title-unread. We are working on the updated coding standards/docs for the front-end, but for now you can reference these http://hanakin.github.io/base-l/#/?id=base-l